DependencyInjection/WhiteOctoberPagerfantaExtension.php

```diff
@@ -50,5 +52,15 @@ public function load(array $configs, ContainerBuilder $container)
             $convertListener = $container->getDefinition('pagerfanta.convert_not_valid_current_page_to_not_found_listener');
             $convertListener->addTag('kernel.event_subscriber');
         }
+
+        // BC layer to inject the 'request' service when RequestStack is unavailable
```

Comment:
I think dropping support for Symfony 2.3 might be fine instead, as it is EOLed

Reply:
It would be a good idea to drop Symfony 2.3 support ... but I think it's better to not do that in this pull request. That way, the people responsible of this bundle can safely merge this without breaking anything. Later, in a separate pull request, they can decide to drop 2.3 support if they want to do that. Thanks!